test/extended/builds/pipeline.go

```diff
@@ -100,6 +95,13 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 	})
 
 	g.Context("Pipeline with maven slave", func() {
+		g.AfterEach(func() {
+			if os.Getenv(jenkins.DisableJenkinsGCStats) == "" {
+				g.By("stopping jenkins gc tracking")
+				ticker.Stop()
+			}
+		})
+
```

Comment:
This review comment is just a question for my enlightenment.  The move of the `g.AfterEach` is interesting.  I'm curious as to how exactly moving within the ginkgo context helped with exiting the test on the failure.

thanks